WalletWasabi.Fluent/ViewModels/Wallets/Home/History/HistoryViewModel.cs

```diff
@@ -119,9 +117,9 @@ public HistoryViewModel(WalletViewModel walletViewModel, IObservable<Unit> updat
 					width: new GridLength(1, GridUnitType.Star)),
 
 				// Incoming
-				new PrivacyTextColumn<HistoryItemViewModelBase>(
+				new TemplateColumn<HistoryItemViewModelBase>(
```

Comment:
The PrivacyTextColumn was specifically used to improve history table performance, especially scrolling perf.